### DISCUSSION TITLE
v1.0.8

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense-companion#848] [Announcements]
## What’s Changed

- Fix coordinates @DTTerastar (#847)

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense-companion/releases/tag/v1.0.8'>v1.0.8</a>.</em>

**Comment by leccelecce** [comment]
Great to see this still being worked on, will be trying out this week!
